### PR TITLE
add check quota

### PR DESCRIPTION
**Pull request by jerry-yu**
add check quota

### CODE REVIEW

**Review by driftluo — DISMISSED** [review]

File: tx_pool/src/pool.rs [hunk 1]
```diff
@@ -150,6 +150,79 @@ impl Pool {
         self.update_order_set(txs);
     }
```
Comment (u2) [inline tx_pool/src/pool.rs]: so, we can remove the `package` method ?

Comment (jerry-yu) [reply tx_pool/src/pool.rs]: we can but not now.  remove when cita-auth modify the use of package

**Review by u2 — DISMISSED** [review]

File: tx_pool/src/pool.rs (hunk 1 above)
Comment (u2) [reply tx_pool/src/pool.rs]: why ?

File: libproto/src/protos/executor.proto [hunk 2]
```diff
@@ -77,6 +77,7 @@ message ConsensusConfig {
     AccountGasLimit account_gas_limit = 2;
     repeated bytes nodes = 3;
     uint64 block_interval = 4;
+    bool check_quota = 5;
```
Comment (rink1969) [inline libproto/src/protos/executor.proto:80]: 这里为什么要加这个信息？

File: tx_pool/src/pool.rs (hunk 1 above)
Comment (jerry-yu) [reply tx_pool/src/pool.rs]: cita-auth can't compile success without package method now

**Review by rink1969 — DISMISSED** [review]

File: libproto/src/protos/executor.proto (hunk 2 above)
Comment (jerry-yu) [reply libproto/src/protos/executor.proto:80]: 为了把信息发给chain，然后chain发给auth

**Review by u2 — APPROVED** [review]

**Review by rink1969 — APPROVED** [review]

**Pull request merged**